Building via image fails as bower won't run with sudo

## Steps to reproduce the issue

csproj contains a `bower install` step:

```xml
  <Target Name="PrepublishScript" BeforeTargets="PrepareForPublish" Condition=" '$(IsCrossTargetingBuild)' != 'true' ">
    <Exec Command="npm install" />
    <Exec Command="bower install" />
    <Exec Command="gulp" />
  </Target>
```

Run a command such as:

```bash
docker run -it -v $(pwd):/app --workdir /app microsoft/aspnetcore-build bash -c "dotnet restore && dotnet publish -c Release -o out"
```

## Expected behavior

Build works.

## Actual behavior

Complains that bower won't run as root, and that bower exit code was 1.

```text
EXEC : Additional error details:  [/app/src/XYZ.Repository/XYZ.Repository.csproj]
  Since bower is a user command, there is no need to execute it with superuser permissions.
  If you're having permission errors when using bower without sudo, please spend a few minutes learning more about how your system should work and make any necessary repairs.

  http://www.joyent.com/blog/installing-node-and-npm
  https://gist.github.com/isaacs/579814

  You can however run a command with sudo using --allow-root option
/app/src/XYZ.Repository/XYZ.Repository.csproj(64,5): error MSB3073: The command "bower install" exited with code 1.
```

## Additional information

Seems a fix is to include the following in the image's Dockerfile:

```
RUN echo '{ "allow_root": true }' > /root/.bowerrc
```

## Output of `dotnet --info`

Build occurs in docker container from 1.1.1 (latest) of this image: https://hub.docker.com/r/microsoft/aspnetcore-build/

## Output of `docker info`

```
$ docker info
Containers: 14
 Running: 2
 Paused: 0
 Stopped: 12
Images: 8
Server Version: 1.12.6
Storage Driver: overlay
 Backing Filesystem: extfs
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: null overlay host bridge
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: apparmor seccomp
Kernel Version: 4.4.0-57-generic
Operating System: Ubuntu 16.04.2 LTS
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 990.6 MiB
Name: ip-172-31-7-161
ID: 6EDU:ZCR5:2SQ4:KJYV:25WS:LA6F:AAOD:YCHQ:BERH:3KW6:D55Q:5BNH
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Insecure Registries:
 127.0.0.0/8
```

--- a/1.0/jessie/sdk/Dockerfile
+++ b/1.0/jessie/sdk/Dockerfile
@@ -30,7 +30,8 @@ RUN buildDeps='xz-utils' \
     && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
     && apt-get purge -y --auto-remove $buildDeps \
     && ln -s /usr/local/bin/node /usr/local/bin/nodejs \
-    && npm install -g bower gulp
+    && npm install -g bower gulp \
+    && echo '{ "allow_root": true }' > /root/.bowerrc
 
 # set up network
 ENV ASPNETCORE_URLS http://+:80

--- a/1.1/jessie/sdk/Dockerfile
+++ b/1.1/jessie/sdk/Dockerfile
@@ -31,7 +31,8 @@ RUN buildDeps='xz-utils' \
     && apt-get purge -y --auto-remove $buildDeps \
     && ln -s /usr/local/bin/node /usr/local/bin/nodejs \
     # set up bower and gulp
-    && npm install -g bower gulp
+    && npm install -g bower gulp \
+    && echo '{ "allow_root": true }' > /root/.bowerrc
 
 # set up network
 ENV ASPNETCORE_URLS http://+:80

--- a/kitchensink/Dockerfile
+++ b/kitchensink/Dockerfile
@@ -30,7 +30,8 @@ RUN buildDeps='xz-utils' \
     && apt-get purge -y --auto-remove $buildDeps \
     && ln -s /usr/local/bin/node /usr/local/bin/nodejs \
     # set up bower and gulp
-    && npm install -g bower gulp
+    && npm install -g bower gulp \
+    && echo '{ "allow_root": true }' > /root/.bowerrc
 
 # Add Docker SDK for when building a solution that has the Docker tools project.
 RUN curl -H 'Cache-Control: no-cache' -o /tmp/Microsoft.Docker.Sdk.tar.gz https://distaspnet.blob.core.windows.net/sdk/Microsoft.Docker.Sdk.tar.gz \

